kogito-codegen/src/main/java/org/kie/kogito/codegen/di/CDIDependencyInjectionAnnotator.java

```diff
@@ -159,17 +158,16 @@ public String emitterType(String dataType) {
         return node;
     }
 
-    @Override
-    public String objectMapperInjectorSource(String packageName) {
-        CompilationUnit clazz = parse(
-                this.getClass().getResourceAsStream("/class-templates/rules/KogitoQuarkusObjectMapper.java"));
-        clazz.setPackageDeclaration( packageName );
-        return clazz.toString();
-    }
-
     @Override
     public <T extends NodeWithAnnotations<?>> T withEagerStartup(T node) {
         node.addAnnotation("io.quarkus.runtime.Startup");
         return node;
     }
+
+    @Override
+    public <T extends NodeWithAnnotations<?>> boolean isRestAnnotated(T node) {
+        return Stream.of("POST", "GET", "PUT", "DELETE")
+                .map(node::getAnnotationByName)
+                .anyMatch(Optional::isPresent);
+    }
```

Comment:
I wonder if

```suggestion
    public <T extends NodeWithAnnotations<?>> boolean isRestAnnotated(T node) {
        return Arrays.asList("POST", "GET", "PUT", "DELETE").contains(node.getAnnotationByName());
    }
```

is "cheaper"

Reply:
Yes it should but `getAnnotationByName` requires a param (the annotation name) so I don't think I can do anything better than iterate over values